helm/templates/configmap.yaml

```diff
@@ -1,21 +1,25 @@
 {{- $chartName := include "job-manager.name" . -}}
+{{- $configmapName := include "configmap.fullname" . }}
 {{- $tracingUrl := include "job-manager.tracingUrl" . -}}
 {{- $metricsUrl := include "job-manager.metricsUrl" . -}}
+{{- $db := (include "common.db.merged" .) | fromYaml }}
+
 {{- if .Values.enabled -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-configmap
+  name: {{ $configmapName }}
```

Comment:
Don't include, just use the chart full name and add "-comfigmap"

Reply:
After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.